lti-tool-provider/src/test/resources/combined-cartridge-basiclti-link.xml

```diff
@@ -0,0 +1,47 @@
+<cartridge_basiclti_link xsi:schemaLocation="http://www.imsglobal.org/xsd/imslticc_v1p0 http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticc_v1p0.xsd
+http://www.imsglobal.org/xsd/imsbasiclti_v1p0 http://www.imsglobal.org/xsd/lti/ltiv1p0/imsbasiclti_v1p0.xsd
+http://www.imsglobal.org/xsd/imslticm_v1p0 http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticm_v1p0.xsd
+http://www.imsglobal.org/xsd/imslticp_v1p0 http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticp_v1p0.xsd" xmlns="http://www.imsglobal.org/xsd/imslticc_v1p0" xmlns:blti="http://www.imsglobal.org/xsd/imsbasiclti_v1p0" xmlns:lticm="http://www.imsglobal.org/xsd/imslticm_v1p0" xmlns:lticp="http://www.imsglobal.org/xsd/imslticp_v1p0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <blti:title>DLR LTI Tool</blti:title>
+  <blti:description>Search or administer DLR resources</blti:description>
+  <blti:extensions platform="canvas.instructure.com">
+    <lticm:property name="tool_id">dlr_combined</lticm:property>
+    <lticm:property name="privacy_level">public</lticm:property>
+    <lticm:property name="domain">unit.no</lticm:property>
+    <lticm:options name="resource_selection">
+      <lticm:property name="icon_url">https://dlr.unit.no/favicon.ico</lticm:property>
+      <lticm:property name="selection_width">800</lticm:property>
+      <lticm:property name="selection_height">800</lticm:property>
+      <lticm:property name="url">https://api.loke.aws.unit.no/lms/canvas/v1/embedExternalTool</lticm:property>
+      <lticm:property name="text">Embed content from the DLR repository</lticm:property>
+      <lticm:property name="enabled">true</lticm:property>
+    </lticm:options>
+    <lticm:options name="editor_button">
+      <lticm:property name="icon_url">https://dlr.unit.no/favicon.ico</lticm:property>
+      <lticm:property name="selection_width">800</lticm:property>
+      <lticm:property name="selection_height">800</lticm:property>
+      <lticm:property name="url">https://api.loke.aws.unit.no/lms/canvas/v1/embedRichContentEditor</lticm:property>
+      <lticm:property name="text">Embed content from the DLR repository</lticm:property>
+      <lticm:property name="enabled">true</lticm:property>
+    </lticm:options>
+    <lticm:options name="course_navigation">
+      <lticm:property name="visibility">public</lticm:property>
+      <lticm:property name="default">enabled</lticm:property>
+      <lticm:property name="url">https://api.loke.aws.unit.no/lms/canvas/v1/site</lticm:property>
+      <lticm:property name="text">DLR</lticm:property>
+      <lticm:property name="enabled">true</lticm:property>
+    </lticm:options>
+  </blti:extensions>
+  <blti:launch_url>https://api.loke.aws.unit.no/lms/canvas/v1/combined</blti:launch_url>
+  <blti:icon>https://dlr.unit.no/favicon.ico</blti:icon>
+  <blti:vendor>
+    <lticp:code>bibsys.no</lticp:code>
+    <lticp:name>BIBSYS</lticp:name>
+    <lticp:contact>
+      <lticp:email>support@bibsys.no</lticp:email>
+    </lticp:contact>
```

Comment:
bibsys?

Reply:
Replika av det vi returnerer i dag. Vi kan opprette ny sak for det om vi ønsker å gjøre slikt konfigurerbart.